src/commands/credits/index.js

```diff
@@ -115,19 +115,26 @@ module.exports = {
   async execute(interaction) {
     if (interaction.options.getSubcommand() === 'balance') {
       await balance(interaction);
-    } else if (interaction.options.getSubcommand() === 'gift') {
+    }
+ else if (interaction.options.getSubcommand() === 'gift') {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 1 space.](https://app.codacy.com/gh/ZynerOrg/xyter/pullRequest?prid=9011014)
```suggestion
		else if (interaction.options.getSubcommand() === 'gift') {
```